### ISSUE TITLE
strange behaviour if allready in paste mode

### ISSUE THREAD

**Issue by DrVanScott** [ConradIrwin/vim-bracketed-paste#17]
sometimes i enable paste mode manually since i have been used to do so for ages ;-)

    :set paste

Assume you have "echo" in the clipboard. After entering input mode, a middle click inserts

    <F28>echo

**Comment by msabramo** [comment]
Yes, I am seeing the same thing. I recorded an asciicast of it to illustrate:

[![asciicast](https://asciinema.org/a/9x64d94s6f6bc2mfxlolv7gu0.png)](https://asciinema.org/a/9x64d94s6f6bc2mfxlolv7gu0)
